scripts/install-shiplog.sh

```
@@ -0,0 +1,122 @@
+
+Options:
+  --force         overwrite existing installation directory
+  --profile FILE  shell profile to update (default: auto-detect)
+  --dry-run       show what would happen without making changes
+  --silent        reduce logging
+  -h, --help      show this help
+USAGE
+}
+
+log() { [ "$SILENT" -eq 1 ] || echo "[shiplog-install] $*"; }
+run() { if [ "$DRY_RUN" -eq 1 ]; then echo "+ $*"; else eval "$@"; fi; }
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)